Documents/BallotDrafts/sarif-v2.1.0-final-e-ballot-1/sarif-external-property-file-schema-2.1.0.json

```diff
@@ -17,13 +17,13 @@
     },
 
     "guid": {
-      "description": "A stable, unique identifer for this external properties object, in the form of a GUID.",
+      "description": "A stable, unique identifier for this external properties object, in the form of a GUID.",
```

Comment:
This is a versioned schema. I presume that this change can't be made, but if it's at all possible to fix going forward, that's the entire reason for this PR -- https://github.com/github/codeql-action/pull/404#discussion_r584257857